deploy/001_deploy.ts

```diff
@@ -44,7 +44,7 @@ const deployFunction: DeployFunction = async function (hre: HardhatRuntimeEnviro
   await deployThroughDeterministicFactory({
     deployer,
     name: 'StaticOracle',
-    salt: 'MF-UniswapV3-StaticOracle-V1',
+    salt: 'MF-UniswapV3-StaticOracle-V1+',
```

Comment:
I'm good with increasing the version, feels like a nice improvement
```suggestion
    salt: 'MF-UniswapV3-StaticOracle-V2',
```